cli/lsp/testing/execution.rs

```diff
@@ -251,6 +252,7 @@ impl TestRun {
     client: &Client,
     maybe_root_uri: Option<&ModuleSpecifier>,
   ) -> Result<(), AnyError> {
+    // todo(dsherret): the LSP should not know about cli flags—use TestOptions here
```

Comment:
I'm not sure if this is correct - in vscode_deno you can specify flags just for testing purposes

Reply:
Yup, you're right.